workspaces/arborist/lib/arborist/index.js

```diff
@@ -78,6 +79,9 @@ class Arborist extends Base {
       lockfileVersion: lockfileVersion(options.lockfileVersion),
       installStrategy: options.global ? 'shallow' : (options.installStrategy ? options.installStrategy : 'hoisted'),
     }
+    if (this.options.installStrategy === 'linked') {
+      this.options.installLinks = false
```

Comment:
if someone has explicitly set the install strategy to linked, and install-links to true, shouldn't this error instead of silently changing their config?